packages/jsii-rosetta/test/translations/imports/submodule-import.py

```diff
@@ -9,7 +9,7 @@
 )
 
 # Accesses two distinct points of the submodule hierarchy
-my_class = calc.submodule.MyClass(prop=calc.submodule.child.SomeEnum.SOME)
+my_class = calc.submodule.MyClass(prop=calc.submodule.child.SomeEnum.SOME, nested_struct=calc.submodule.child.AnotherStruct(string_property="hello"))
```

Comment:
This currently fails as the result from Rosetta is missing the `calc.submodule.child.` for `AnotherStruct`
<img width="962" alt="image" src="https://user-images.githubusercontent.com/1112056/231726413-030e242c-fb83-4496-a0ca-ad1a5cc7fdf9.png">